gc/base/standard/ConcurrentGC.cpp

```diff
@@ -950,30 +684,29 @@ MM_ConcurrentGC::resetInitRangesForSTW()
  * @return TRUE if work left to be done, FALSE otherwise.
  */
 bool
-MM_ConcurrentGC::getInitRange(MM_EnvironmentBase *env, void **from, void **to, InitType *type, bool *concurrentCollectable )
+MM_ConcurrentGC::getInitRange(MM_EnvironmentBase *env, void **from, void **to, InitType *type, bool *concurrentCollectable)
 {
     uint32_t i;
-    void *localTo,*localFrom;
+    void *localTo, *localFrom;
     uint32_t oldIndex, newIndex;
```

Comment:
move down declarations to the points they are used first time, or if you need to keep this scope, then initialize them with a neutral value